components/01-globals/blocquote/Blockquote.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <blockquote>
+    {{ blockquote }}
```

Comment:
Content should be passed via slot, we don't want to limit users in any way, nor use `v-html`